### PR TITLE
TD 2658 Expose Ref for Wizard Content

### PR DESCRIPTION
**Pull request by jegasriskantha**
Contributes to [TD-2658](https://sce.myjetbrains.com/youtrack/issue/TD-2658/Create-Step-3)

## Changes

Updates the WizardContent to pass on a ref to the underlying box component. This is done so that the scroll postion of the scroll bar in the WizardContent can be set. 

## Dependencies

N/A

## UI/UX

N/A

## Testing notes

- Not much to test here, just check that the ref forwarding looks correct. 

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [x] Branch has been run in docker.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [ ] Lint and test workflows pass.

### CODE REVIEW

**Comment by jegasriskantha** [comment]
Tests are failing due to flaky playwright tests

File: src/Wizard/WizardContent/WizardContent.tsx [hunk 1]
```diff
@@ -3,9 +3,13 @@ import * as React from "react";
 import { Box } from "@mui/material";
 import { WizardContentProps } from "./WizardContent.types";
 
-export default function WizardContent({ children }: WizardContentProps) {
+export default React.forwardRef(function WizardContent(
```
Comment (syedsalehinipg) [inline src/Wizard/WizardContent/WizardContent.tsx:6]: image


I think we should add type for ref for type safety
```suggestion
export default React.forwardRef<HTMLDivElement>(function WizardContent(
```

**Review by syedsalehinipg — CHANGES_REQUESTED** [review]
There seems to be a conflict with the `package-lock.json` if you can fix that please,

File: src/Wizard/WizardContent/WizardContent.tsx [hunk 2]
```diff
@@ -3,9 +3,13 @@ import * as React from "react";
 import { Box } from "@mui/material";
 import { WizardContentProps } from "./WizardContent.types";
 
-export default function WizardContent({ children }: WizardContentProps) {
+export default React.forwardRef<HTMLDivElement, WizardContentProps>(
+  { children }: WizardContentProps,
+  ref
+) {
```
Comment (syedsalehinipg) [inline src/Wizard/WizardContent/WizardContent.tsx:9]: I seem to get an eslint error:

image

The suggestion fixes it:

```suggestion
export default React.forwardRef<HTMLDivElement, WizardContentProps>(
  function WizardContent({ children }, ref) {
```

**Review by syedsalehinipg — APPROVED** [review]

**Pull request closed without merging**